packages/react-aria-components/stories/Table.stories.tsx

```diff
@@ -506,11 +796,163 @@ export function VirtualizedTableWithEmptyState() {
+        items: json.results,
+        cursor: json.next
+      };
+    }
+  });
+
+  let layout = useMemo(() => {
+    return new TableLayout({
+      rowHeight: 25,
+      headingHeight: 25
+    });
+  }, []);
+
+  let isLoading = list.loadingState === 'loading' || list.loadingState === 'loadingMore';
+  let scrollRef = useRef<HTMLTableElement>(null);
+  let memoedLoadMoreProps = useMemo(() => ({
+    isLoading: isLoading,
+    onLoadMore: list.loadMore,
+    items: list.items
+  }), [isLoading, list.loadMore, list.items]);
+  useLoadMore(memoedLoadMoreProps, scrollRef);
+
+  return (
+    <Virtualizer layout={layout}>
+      <Table aria-label="Load more table virtualized" ref={scrollRef} style={{height: 150, width: 400, overflow: 'auto'}}>
+        <TableHeader style={{background: 'var(--spectrum-gray-100)', width: '100%', height: '100%'}}>
+          <Column id="name" isRowHeader>Name</Column>
+          <Column id="height">Height</Column>
+          <Column id="mass">Mass</Column>
+          <Column id="birth_year">Birth Year</Column>
+        </TableHeader>
+        <MyTableBody
+          renderEmptyState={() => renderEmptyLoader({isLoading: list.loadingState === 'loading'})}
+          isLoadingMore={list.loadingState === 'loadingMore'}
+          rows={list.items}>
+          {(item) => (
+            <Row id={item.name} style={{width: 'inherit', height: 'inherit'}}>
+              <Cell>{item.name}</Cell>
+              <Cell>{item.height}</Cell>
+              <Cell>{item.mass}</Cell>
+              <Cell>{item.birth_year}</Cell>
+            </Row>
+          )}
+        </MyTableBody>
+      </Table>
+    </Virtualizer>
+  );
+};
+
+export const OnLoadMoreTableStoryVirtualized  = {
+  render: OnLoadMoreTableVirtualized,
+  name: 'Virtualized Table with async loading'
+};
+
+const OnLoadMoreTableVirtualizedResizeWrapper = () => {
+  let list = useAsyncList<Character>({
+    async load({signal, cursor}) {
+      if (cursor) {
+        cursor = cursor.replace(/^http:\/\//i, 'https://');
+      }
+
+      // Slow down load so progress circle can appear
+      await new Promise(resolve => setTimeout(resolve, 4000));
+      let res = await fetch(cursor || 'https://swapi.py4e.com/api/people/?search=', {signal});
+      let json = await res.json();
+      return {
+        items: json.results,
+        cursor: json.next
+      };
+    }
+  });
+
+  let layout = useMemo(() => {
+    return new TableLayout({
+      rowHeight: 25,
+      headingHeight: 25
+    });
+  }, []);
+
+  let isLoading = list.loadingState === 'loading' || list.loadingState === 'loadingMore';
+  let scrollRef = useRef<HTMLDivElement>(null);
+  let memoedLoadMoreProps = useMemo(() => ({
+    isLoading: isLoading,
+    onLoadMore: list.loadMore,
+    items: list.items
+  }), [isLoading, list.loadMore, list.items]);
+  useLoadMore(memoedLoadMoreProps, scrollRef);
+
+  return (
+    <ResizableTableContainer ref={scrollRef} style={{height: 150, width: 400, overflow: 'auto'}}>
```

Comment:
columns of this story don't appear to be resizable?

Reply:
they are not, more of a test that the scroll ref could be attached to this wrapper and still works with load more.